README.md

```diff
@@ -63,34 +65,44 @@ However, we're proposing unlimited-precision decimal instead, for the following
+    - There is no Infinity, -0, NaN, etc; error cases lead to exceptions, just like BigInt, and `-0d` is `0d` ([#9](https://github.com/littledan/proposal-bigdecimal/issues/9))
+- A new primitive type, not an object: `typeof 1d === "bigdecimal"`
+    - There can still be methods on `BigDecimal.prototype` due to the magic of wrappers, just like Number.
+
+Operator semantics:
+- Operators which can be calculated exactly are defined to return their exact answer (`+`, `-`, `*`, `%`, etc.) ([#10](https://github.com/littledan/proposal-bigdecimal/issues/10))
+- Operators which would need to round instead have methods to support them, described below ([#13](https://github.com/littledan/proposal-bigdecimal/issues/13))
+- Bitwise operators are not supported, as they don't logically make sense on the BigDecimal domain ([#20](https://github.com/littledan/proposal-bigdecimal/issues/20))
+- Use explicit casts when you need to do a calculation involving different numerical types. Otherwise, a TypeError is thrown, like for BigInt+Number.
+- Comparison with `===` compares two BigDecimals for mathematical equality, and returns false if comparison is with another type; comparison with `==`, `<`, etc can compare BigDecimal with any numerical type
+
+BigDecimal methods for calculation: ([#14](https://github.com/littledan/proposal-bigdecimal/issues/14))
+- `BigDecimal.prototype.round()` rounds a BigDecimal, based on an options bag with the following parameters:
+    - `roundingMode`: Rounding mode, with exact set of values TBD, maybe including `"up"`, `"down"`, `"half-up"`, `"half-down"`, `"half-even"` (more?). There is no default; this must be explicitly provided
+    - Exactly one of the two following options is required to indicate the precision to round to (names matching Intl.NumberFormat):
+        - `maximumFractionDigits`: The maximum number of decimal places after the `.`
+        - `maximumSignificantDigits`: The maximum number of significant digits
+- `BigDecimal.prototype.div`, `BigDecimal.prototype.pow`: Takes two parameters: a BigDecimal (for the second operand) and a rounding mode
+    - E.g., `1m.div(3m, { maximumFractionDigits: 2, roundingMode: "down" }) === .33m`
+- `BigDecimal.prototype.partition(pieces, roundingOptions)` returns an Array of length `pieces` with the BigDecimal split as evenly as possible, based on the rounding options which indicate precision
+- BigDecimal64Array and BigDecimal128Array (binary format implementation-defined to be one of the two IEEE formats, and then dataview methods take flag; ([#16](https://github.com/littledan/proposal-bigdecimal/issues/16)))
+- Possible other methods: divmod? quantum? compareTotal? significantDigits/fractioDigits? sqrt? trig fns? (#xxx)
+
+BigDecimal methods for string formatting:
+- `BigDecimal.prototype.toString()` is similar to the behavior on Number, e.g., `123.456d.toString()` is `"123.456"`. ([#12](https://github.com/littledan/proposal-bigdecimal/issues/12))
```

Comment:
So `123.d.toString()` (along with `123.d`) is a valid syntax also? I think we should clarify that, and as I remember, @ljharb don't like this idea.

Reply:
Yeah, let's discuss these cases further in #7 . I think those two cases are a little different from each other; I'd be OK with banning `1.d` while permitting `1.0d.toString()`, personally.